src/Orleans.CodeGeneration.Build/Program.cs

```diff
@@ -132,7 +133,8 @@ public static int Main(string[] args)
                     Console.WriteLine("ERROR: Orleans-CodeGen - the input assembly does not reference Orleans and therefore code can not be generated.");
                     return -2;
                 }
-
+                stopWatch.Stop();
+                Console.WriteLine($"Built time code generation for assembly {options.InputAssembly} took {stopWatch.ElapsedMilliseconds} miliseconds");
```

Comment:
s/Built/Build
s/miliseconds/milliseconds
I think `ms` as a suffix would be fine, too

Reply:
My English teacher is crying in the corner...